cli/cmd/flags.go

```diff
@@ -64,19 +66,38 @@ func bindRPCURL(cmd *cobra.Command, rpcURL *string) {
 func bindUnjailConfig(cmd *cobra.Command, cfg *unjailConfig) {
 	netconf.BindFlag(cmd.Flags(), &cfg.Network)
 	bindPrivateKeyFile(cmd, &cfg.PrivateKeyFile)
-	_ = cmd.MarkFlagRequired("network")
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindDelegateValConfig(cmd *cobra.Command, cfg *delegateValConfig) {
+	netconf.BindFlag(cmd.Flags(), &cfg.network)
+	bindPrivateKeyFile(cmd, &cfg.privateKeyFile)
+
+	cmd.Flags().Uint64Var(&cfg.amount, flagDelegationAmount, cfg.amount, "increase delegation amount in OMNI (minimum 1 OMNI)")
+
+	bindExecutionRPC(cmd, &cfg.evmRPC)
```

Comment:
lets create a `bindEOAConfig` function that is shared between all commands that embed eoaConfig.